### ISSUE TITLE
Update for React Native 0.61.5

### ISSUE THREAD

**Issue by sagargulati** [rogerwilliamsio/react-native-gpgs#1]
Are we looking to maintain this package? Looking for future updates and changes.

**Comment by sagargulati** [comment]
@rogerwilliamsio anything?

**Comment by MariusCatanoiu** [comment]
same here. "Sign In Failed"

**Comment by sagargulati** [comment]
@rogerwilliamsio : Can you merge the pull, it should fix the issues? https://github.com/rogerwilliamsio/react-native-gpgs/pull/2
https://github.com/rogerwilliamsio/react-native-gpgs/pull/3

**Comment by MariusCatanoiu** [comment]
> @rogerwilliamsio : Can you merge the pull, it should fix the issues? #2
> #3

for me, this didn't fix the issue :(

**Comment by sagargulati** [comment]
@MariusCatanoiu : please make sure you have SHA1 for your key which you use to sign the APK added in your cloud console or firebase.

**Comment by MariusCatanoiu** [comment]
> @MariusCatanoiu : please make sure you have SHA1 for your key which you use to sign the APK added in your cloud console or firebase.

yes, I have it :(

**Comment by sagargulati** [comment]
@MariusCatanoiu : I can bet on this, you might have the wrong SHA1 signature ID in the cloud console. I had the same issues, it would work fine even without the update. Also, did you followed the https://developers.google.com/games/services/android/quickstart steps?

**Comment by MariusCatanoiu** [comment]
> @MariusCatanoiu : I can bet on this, you might have the wrong SHA1 signature ID in the cloud console. I had the same issues, it would work fine even without the update. Also, did you followed the https://developers.google.com/games/services/android/quickstart steps?

Yes, is did that. I implemented the GPGS in a native android test app. And works fine.
But on RN doesn't seem to work the login

**Comment by sagargulati** [comment]
Can you create expo and show the example?

**Comment by MariusCatanoiu** [comment]
> Can you create expo and show the example?

I found a fix. I updated the build.gradle from the package with compileSdkVersion 28 (min version needed for the current GPGS api) and removed requestScopes(Drive.SCOPE_APPFOLDER) from RNGPGSAuth.

now the login works :D

**Comment by sagargulati** [comment]
That's what my pull request is about.

**Comment by DanielePetrarolo** [comment]
Hello guys, as today and with the latest RN version, is this package working fine?

**Comment by MariusCatanoiu** [comment]
Hey Daniele. I don't think so.
On my code I updated the SDK from ZendeskSDK to ZendeskSupportSDK.
android gradle dependency com.zendesk 4.0.0, compileSdkVersion 28 (for android) and swift v to 5.1.2 (for ios)
After that works fine, both platforms.
And of course I edited the package code for my needs (for example, I added to send custom fields)
I have RN 0.62.2

**Comment by DanielePetrarolo** [comment]
I just need Google Play Games login, leaderboard and achievement unlock. Can this package work with my lates RN version?

**Comment by MariusCatanoiu** [comment]
without changes on the module code, nope

**Comment by Insullone** [comment]
> I found a fix. I updated the build.gradle from the package with compileSdkVersion 28 (min version needed for the current GPGS api) and removed requestScopes(Drive.SCOPE_APPFOLDER) from RNGPGSAuth.
> 
> now the login works :D
@MariusCatanoiu I followed your steps but have seen a new error when I show leaderboard:
[Error: 17: API: Games.API is not available on this device.]
This error does not show up at first, but it occurs after running 'npx react-native run-android' several times. Have you seen the same issue? By the way, how do you change the Java code & build.gradle locally, since your fix is not merged in master. Did you use patch or create your own package with the fix, to make sure the original package is not used? Thank you.

**Comment by MariusCatanoiu** [comment]
> > I found a fix. I updated the build.gradle from the package with compileSdkVersion 28 (min version needed for the current GPGS api) and removed requestScopes(Drive.SCOPE_APPFOLDER) from RNGPGSAuth.
> > now the login works :D
> 
> Hi @MariusCatanoiu I followed your steps but have seen a new error when I show leaderboard:
> [Error: 17: API: Games.API is not available on this device.]
> This error does not show up at first, but it occurs after running 'npx react-native run-android' several times. Have you seen the same issue? By the way, how do you change the Java code & build.gradle locally, since your fix is not merged in master. Did you use patch or create your own package with the fix, to make sure the original package is not used? Thank you.

no. you have the Google Drive API enabled? usually the error appears if you have Google Drive API disabled
I change java code in node modules and use patch-package to save changes.
